content/en/docs/tasks/traffic-management/egress/egress-gateway-tls-origination/index.md

```diff
@@ -398,248 +388,55 @@ to hold the configuration of the NGINX server:
     EOF
     {{< /text >}}
 
-1.  Define a `ServiceEntry` and a `VirtualService` for `nginx.example.com` to instruct Istio to direct traffic destined
-    to `nginx.example.com` to your NGINX server:
-
-    {{< text bash >}}
-    $ kubectl apply -f - <<EOF
-    apiVersion: networking.istio.io/v1alpha3
-    kind: ServiceEntry
-    metadata:
-      name: nginx
-    spec:
-      hosts:
-      - nginx.example.com
-      ports:
-      - number: 80
-        name: http
-        protocol: HTTP
-      - number: 443
-        name: https
-        protocol: HTTPS
-      resolution: DNS
-      endpoints:
-      - address: my-nginx.mesh-external.svc.cluster.local
-        ports:
-          https: 443
-    ---
-    apiVersion: networking.istio.io/v1alpha3
-    kind: VirtualService
-    metadata:
-      name: nginx
-    spec:
-      hosts:
-      - nginx.example.com
-      tls:
-      - match:
-        - port: 443
-          sniHosts:
-          - nginx.example.com
-        route:
-        - destination:
-            host: nginx.example.com
-            port:
-              number: 443
-          weight: 100
-    EOF
-    {{< /text >}}
-
-#### Deploy a container to test the NGINX deployment
+1.  Deploy the [sleep]({{< github_tree >}}/samples/sleep) sample to test sending requests to the NGINX server:
+    
+        If you have enabled [automatic sidecar injection](/docs/setup/additional-setup/sidecar-injection/#automatic-sidecar-injection), do
+    
+        {{< text bash >}}
+        $ kubectl apply -f @samples/sleep/sleep.yaml@
+        {{< /text >}}
+    
+        otherwise, you have to manually inject the sidecar before deploying the `sleep` application:
+    
+        {{< text bash >}}
+        $ kubectl apply -f <(istioctl kube-inject -f @samples/sleep/sleep.yaml@)
+        {{< /text >}}
+    
+        Note that any pod that you can `exec` and `curl` from would do.
 
-1.  Create Kubernetes [Secrets](https://kubernetes.io/docs/concepts/configuration/secret/) to hold the client's and CA
```

Comment:
Here you remove the testing the NGINX server. I like to provide a test for each step to verify that each step works. This makes the tasks long though.

Reply:
I purpose a different approach to this then, how about we do a SIMPLE TLS origination with CaCert from sleep first, show that it fails, then update the DR to do a MUTUAL TLS and show it succeeds? I think this is more in line with what this user task is for

Reply:
That could be OK, it is an indirect test. I just like to show how you can perform the test directly from a container. Mounting certificates to a container and performing Simple/Mutual TLS is also useful for Istio users in general. Agree that it makes the task longer.

Reply:
I'm OK with shortening this doc because it is particularly long. If the ngnix server isn't working, the user will find out in the next step. It wouldn't be a bad idea to have another security task that shows how to configure a sidecar to directly call an mtls server. If that task existed, and used the same nginx example, we could add a `{{< tip >}}` here that says you can test that the nginx server is working by following those referenced instructions.